tms_objects2.erb

```diff
@@ -177,21 +177,45 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
           </lido:repositoryLocation>
         </lido:repositorySet>
       </lido:repositoryWrap>
-	<% unless State.nil? && Edition.nil? && CatRais.nil? %>
+        <% unless State.nil? && Edition.nil? && CatRais.nil? %>
       <lido:displayStateEditionWrap>
         <% unless State.nil? %>
           <lido:displayState><%= State %></lido:displayState>
         <% end %>
         <% unless Edition.nil? %>
           <lido:displayEdition><%= Edition %></lido:displayEdition>
         <% end %>
-	<% unless CatRais.nil? %>
+        <% unless CatRais.nil? %>
           <lido:sourceStateEdition><%= CatRais %></lido:sourceStateEdition>
         <% end %>
       </lido:displayStateEditionWrap>
-	<% end %>
+        <% end %>
       <lido:objectMeasurementsWrap>
-    </lido:objectIdentificationWrap>
+        <% ObjDimElem.each do |e| %>
+          <lido:objectMeasurementSet>
+            <lido:displayObjectMeasurements><%= e["DisplayDimensions"] %>
+            </lido:displayObjectMeasurements>
+            <lido:objectMeasurements>
+        <% ObjDim.each do |d| %>
+        <% if d["ElementID"] == e["ElementID"] %>
+              <lido:measurementsSet>
+                <lido:measurementType><%= d["DimensionType"] %></lido:measurementType>
+                <lido:measurementUnit><%= d["UnitLabel"] %></lido:measurementUnit>
+                <lido:measurementValue><%= d["DMcm"] %></lido:measurementValue>
+              </lido:measurementsSet>
+        <% end %>
+        <% end %>
+        <% if e["element"] == 'Support (PTG)' %>
+              <lido:extentMeasurements></lido:extentMeasurements>
+        <% else %>
+              <lido:extentMeasurements><%= e["element"] %></lido:extentMeasurements>
+        <% end %>
+              <lido:qualifierMeasurements><%= e["Description"] %></lido:qualifierMeasurements>
```

Comment:
so none of the top 10 objects in the test set implement this, so have to explicitly test this.  

I could tell because I ran "grep qualifierMeasurements ./*" and none of the top 10 have it

Hint: run the view Coboat_ObjectDimElement, find an ID where Description isn't null, and add a line like: 

https://github.com/ycba-cia/TMSSerialization/blob/main/tms_objects2.erb#L204

Run with that line like 204 instead of line 203, and check output for description, when done, I'd suggest keeping the line 204 but commenting it out before committing